pkg/tools/etcdtest.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tools
+
+import (
+	"os"
+	"path"
+)
+
+func PathPrefix() string {
+	return os.Getenv("ETCD_PREFIX")
+}
+
+func AddPrefix(in string, h EtcdHelper) string {
+	return path.Join("/", PathPrefix(), in)
```

Comment:
suggest leaving out the "/" in front-- should use exactly what PathPrefix() returns. (or add it to PathPrefix...)